### PR TITLE
Product categories

### PR DESCRIPTION
**Pull request by svet-ko**

### CODE REVIEW

File: models/category.ts [hunk 1]
```diff
@@ -0,0 +1,68 @@
+import { Category } from "../types/categories.js";
+
+export class CategoryRepo {
+  categories: Category[] = [
+    {
```
Comment (svet-ko) [inline models/category.ts:6]: Doesn't look like categories names. It's better to change)

File: models/category.ts [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+      description: "High-performance laptop for all your needs.",
+    },
+    {
+      id: 2,
+      name: "Smartphone",
+      description: "Latest smartphone with advanced features.",
+    },
+  ];
+
+  generateNewId() {
```
Comment (svet-ko) [inline models/category.ts]: Move this function to /utils/generateNewId.ts
It should not be method of this class

File: models/category.ts (hunk 1 above)
Comment (mdtouhidulislam136) [reply models/category.ts:6]: Thanks, Done

File: models/category.ts (hunk 2 above)
Comment (mdtouhidulislam136) [reply models/category.ts]: Thanks, Done

**Pull request merged**